packages/react-saasify/src/themes/waves.module.css

```diff
@@ -0,0 +1,598 @@
+:root {
+  /* TODO add postcss to resolve vars for IE11 */
+  --color-primary: #5061cb;
+  --filter-tint: saturate(102%) hue-rotate(10deg);
+  --color-fg: #3a3a3a;
+  --max-width: 1108px;
+  --max-width-half: 554px;
+  --space-xs: 4px;
+  --space-s: 8px;
+  --space-m: 16px;
+  --space-l: 24px;
+  --space-xl: 32px;
+  --space-xxl: 48px;
+  --space-xxxl: 64px;
+  --side-gutter-half: var(--space-m);
+  --side-gutter: var(--space-xl);
+  --font-family: 'Avenir Next';
+
+  --line-height-m: 1.375;
+  --line-height-l: 1.42;
+}
+
+.theme-waves {
+  color: #fff;
+  font-family: var(--font-family);
+  font-size: 16px;
+}
+
+.theme-waves * {
+  transition: all 250ms ease;
```

Comment:
interesting && aggressive 😄

Reply:
Yup, overly aggressive. Going to scope it down.

Reply:
Scoped down.